cpp/oneapi/dal/algo/kmeans/common.cpp

```diff
@@ -20,6 +20,30 @@
 
 namespace oneapi::dal::kmeans {
 namespace detail {
+
+result_option_id get_compute_assignments_id() {
```

Comment:
So are all these result options capable of being set in sklearnex KMeans args?

Reply:
yes